src/app/modules/main/chat_section/controller.nim

```diff
@@ -252,7 +252,9 @@ proc setActiveItemSubItem*(self: Controller, itemId: string, subItemId: string)
   self.activeItemId = itemId
   self.activeSubItemId = subItemId
 
-  self.messageService.asyncLoadInitialMessagesForChat(self.getActiveChatId())
+  let chatId = self.getActiveChatId()
+  if chatId == "":
+    self.messageService.asyncLoadInitialMessagesForChat(chatId)
```

Comment:
:eyes: this logic looks super suspicious -> "when chat id is empty load messages for it"

Reply:
:facepalm:

Reply:
Fixed